### PR TITLE
Adding Prereq

### PR DESCRIPTION
**Pull request by TokyoScarab**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4494

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/set-up-audio-conferencing-in-teams.md [hunk 1]
```diff
@@ -34,7 +34,10 @@ For frequently asked questions about Audio Conferencing, see [Audio Conferencing
 
 > [!NOTE]
 > [!INCLUDE [updating-admin-interfaces](includes/updating-admin-interfaces.md)]
```
Comment (JohanFreelancer9) [inline Teams/set-up-audio-conferencing-in-teams.md:36]:
```suggestion
> We are frequently updating how you can manage features, so the steps here might be a little different from what you see.
```

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tonysmit** [comment]
**@scanum - please don't merge this.** @TokyoScarab - what you are saying may be true for SFBO (which I don't believe it is either), but it's not correct for Teams. We don't support 3rd party AC providers in Teams. I just tested it; Assigned a license, then looked at the user's properties. There is no way to change the AC provider in Teams. This has been the case for several years now.

**Pull request closed without merging**